### ISSUE TITLE
Round report can get wonky with reporting

### ISSUE THREAD

**Issue by Myctai** [space-wizards/space-station-14#9936]
## Description
Round report can get wonky with reporting. Issue #9935 is not the single thing adminbus can cause. Adminbus from previous round somehow carried over to my round report! Don't have the round ID, but first I played as latejoin musician, only to be turned into syndie few minutes in with 3 kill objectives + glorious. There were syndies, nukies and zombies. A mess. Round 12611 was the one for which screenshots are made. Here I latejoined as musician again, got syndie'd again, this time it's stay alive, protect and jetpack theft objectives.

At the end, report contains two entries for me! How the hell did my greentext (that did not even show in Secret extended round report with zombie traitor nukies mess) carry over?

**Reproduction**
- Adminbus
- Then adminbus harder?..

**Screenshots**
(I am Lindy Young on both screenshots of same round report)
![12611_1](https://user-images.githubusercontent.com/108953437/180099945-fe96c8f9-bd63-4d23-9d13-8c7ef7a3d4b9.png)
![12611_2](https://user-images.githubusercontent.com/108953437/180099954-32118fc1-3620-45d5-b129-ad4100d2096f.png)

**Additional context**
what is this i don't even

**Comment by EmoGarbage404** [comment]
This just seems like admin stuff?

**Comment by EmoGarbage404** [comment]
I dont understand what this issue is even referring to

**Comment by drakewill-CRL** [comment]
(that did not even show in Secret extended round report with zombie traitor nukies mess)

 If the first round didn't report, that means the rule wasn't on, and the list of traitors probably didn't clear out because the mode wasn't picked (and therefor Ended() wasn't called). The second round DID roll traitors for the mode, and that caused the report to occur, with a very long list of traitors from both rounds.

**Comment by PaulRitter** [comment]
yep, this is an issue with the "Make Traitor"-Verb not enabling the traitor gamemode, hence the list of traitors not getting cleared at the end of the round.

cc @moonheart08

**Comment by lzk228** [comment]
so what is the issue?
